display_list/effects/dl_image_filter.h

```diff
@@ -229,11 +229,13 @@ class DlBlurImageFilter final : public DlImageFilter {
   static std::shared_ptr<DlImageFilter> Make(SkScalar sigma_x,
                                              SkScalar sigma_y,
                                              DlTileMode tile_mode) {
-    if (SkScalarIsFinite(sigma_x) && sigma_x > SK_ScalarNearlyZero &&
-        SkScalarIsFinite(sigma_y) && sigma_y > SK_ScalarNearlyZero) {
-      return std::make_shared<DlBlurImageFilter>(sigma_x, sigma_y, tile_mode);
+    if (!SkScalarIsFinite(sigma_x) || !SkScalarIsFinite(sigma_y)) {
+      return nullptr;
     }
-    return nullptr;
+    if (sigma_x < SK_ScalarNearlyZero && sigma_y < SK_ScalarNearlyZero) {
```

Comment:
Perhaps also clip the values against NearlyZero rather than just checking if they are smaller, so that if we do make a filter out of them, the filter won't have negative sigmas...?

Reply:
done